pymove/_version.py

```diff
@@ -110,34 +105,27 @@ def run_command(
 
 
 def versions_from_parentdir(parentdir_prefix, root, verbose):
-    """
-    Try to determine the version from the parent directory name.
+    """Try to determine the version from the parent directory name.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5497268)